cobol.md

```diff
@@ -0,0 +1,234 @@
+---
+title:COBOL
+description:COBOL is used for writing application programs and we cannot use it to write system software. The applications like those in defense domain, insurance domain, etc. which require huge data processing make extensive use of COBOL.
+created: 1959-04-25
+updated: 2014-09-30
+---
+
+### Installation
+> For Windows and Linux Users
+|Link|Description|
+|---|---|
+|`http://downloads.hercules-390.eu/hercules-3.13.zip`|For Windows Users,This is a Mainframe Emulator for writing programs in COBOL|
+|`http://downloads.hercules-390.eu/hercules-3.13.tar.gz`|For Linux Users,This is a Mainframe Emulator for writing programs in COBOL|
+
+### Examples
+
+```cobol
+IDENTIFICATION DIVISION.
+PROGRAM-ID. hello 10 times.
+PROCEDURE DIVISION.
+PERFORM 10 TIMES
+   DISPLAY "Hello"
+END-PERFORM
+STOP RUN
+```
+## Data types
+
+|Value Type| Description|
+|-----|-----|
+|Data Name| refer to the storage space in the memory where the actual value is stored |
+|Level Number| The level number specifies the hierarchy or levels of data entries/items.It is a a one-digit or two-digit integer between 01 and 49, or one of three special level-numbers: 66, 77, or 88|
+|Picture Clause|It is used to specify the type and size of an elementary data item.The Picture character-string may contain a maximum of 90 characters. It consists of certain COBOL characters used as symbols|
+|Value Clause|Value clause is an optional clause which is used to initialize the data items. The values can be numeric literal, alphanumeric literal, or figurative constant. It can be used with both group and elementary items.|
+
+
+##Basic Verbs
```

Comment:
Please fix the markdown formatting. 
ex. there should be a space after ##

Reply:
fixed that @onecompiler-ops